java/opendcs/src/main/java/decodes/db/NetworkList.java

```diff
@@ -214,6 +215,11 @@ public boolean equals(Object ob)
 		if (!siteNameTypePref.equalsIgnoreCase(nl.siteNameTypePref))
 			return false;
 
+		if (this.networkListEntries.size() != nl.networkListEntries.size())
```

Comment:
Looking at the code below. Is is possible for these to be the same size but have 

A is a subset of B
B is not a subset of A.

I'm pretty sure now, but Set Theory was never my strong suit.